starknet_py/tests/unit/signer/speculos_automation.json

```diff
@@ -2,10 +2,13 @@
   "version": 1,
   "rules": [
     {
-      "text": "Confirm Hash to sign",
-      "conditions": [],
+      "text": "Blind igning",
```

Comment:
note: It's not a typo - for some reason that's how ledger display the text

![Uploading image.png…]()